src/test/java/Test5.java

```diff
@@ -0,0 +1,32 @@
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.List;
+
+public class Test5 {
+
+@Test
+    public void test() {
+    WebDriver driver = new ChromeDriver();
+    driver.get("https://demoqa.com/automation-practice-form/");
+    driver.manage().window().maximize();
+    List<WebElement> oCheckBox = driver.findElements(By.xpath("//*[@class='custom-control-label']"));
+    for (int i = 0; i < oCheckBox.size(); i++) {
+        String sValue = oCheckBox.get(i).getText();
+        if (sValue.equals("Female")) oCheckBox.get(i).click();
+        if (sValue.equals("Music")) oCheckBox.get(i).click();
```

Comment:
would therr makes sense to have also 'else' clause in order to inform that no action was performed?

Reply:
In case if there isn`t such an element.  Got it